be/src/common/config.h

```diff
@@ -958,6 +958,9 @@ DECLARE_Int32(segcompaction_small_threshold);
 // enable java udf and jdbc scannode
 DECLARE_Bool(enable_java_support);
 
+// jdbc connect max wait time
+DECLARE_Int32(jdbc_max_wait_time);
```

Comment:
In the past, user could set -DJDBC_MIN_POOL=1 -DJDBC_MAX_POOL=100 -DJDBC_MAX_IDEL_TIME=300000 , do you forget to set JDBC_MAX_POOL as config?